spec/units/transcoding_spec.rb

```diff
@@ -25,7 +25,7 @@ class GenericFile < ActiveFedora::Base
         when 'video/avi'
           obj.transform_file :content, { mp4: { format: 'mp4' }, webm: { format: 'webm'}, thumbnail: { format: 'jpg', datastream: 'thumbnail' } }, processor: :video
         when 'image/png', 'image/jpg'
-          obj.transform_file :content, { medium: "300x300>", thumb: "100x100>", access: { format: 'jpg', datastream: 'access'} }
+          obj.transform_file :content, { medium: "300x300>", thumb: "100x100>", access: { format: 'jpg', datastream: 'access'} }, processor: 'image'
```

Comment:
Do we need to change this test?  It was checking that image was the default. Are you changing that?